vtex-apps/my-account-commons
# Fix last status not concluding correctly and add pickup point deliver…

…y channel handling

#### What is the purpose of this pull request?

Title is self-explanatory.

#### What problem is this solving?

There were no handling for pickup points.

#### How should this be manually tested?

Make an order with pick up point delivery channel and invoice it in the OMS. After that, check the progress bar, the last status should be "Ready for pick up".

#### Screenshots or example usage

now:
![image](https://user-images.githubusercontent.com/11592617/62630235-ea189c80-b904-11e9-8a22-1e4af4b30751.png)
before:
![image](https://user-images.githubusercontent.com/11592617/62630243-ee44ba00-b904-11e9-8138-901048938768.png)


#### Types of changes
